overlord/snapstate/backend/setup.go

```diff
@@ -111,6 +114,67 @@ func (b Backend) SetupSnap(snapFilePath, instanceName string, sideInfo *snap.Sid
 	return t, installRecord, nil
 }
 
+func earlyKernelMountDir(ksnapName string, rev snap.Revision) string {
+	return filepath.Join("/run/mnt/kernel-snap", ksnapName, rev.String())
```

Comment:
shouldn't this use kernel-snaps? or kernels?

Reply:
Changed